db/config.cc

```diff
@@ -382,6 +382,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "Log a warning when writing cells larger than this value")
     , compaction_rows_count_warning_threshold(this, "compaction_rows_count_warning_threshold", value_status::Used, 100000,
         "Log a warning when writing a number of rows larger than this value")
+    , compaction_collection_items_count_warning_threshold(this, "compaction_collection_items_count_warning_threshold", value_status::Used, 10000,
+        "Log a warning when writing a collection containing more items than this value")
```

Comment:
Let's make this LiveUpdate so a user can change the threshold without restarting the server.

Reply:
We can do this as a follow-up, since related config items also need changing.

Reply:
https://github.com/scylladb/scylladb/issues/11685